src/Components/Nav.js

```diff
@@ -13,16 +53,40 @@ const Nav = () => {
             <SearchBar type="text" placeholder="가고 싶은 역을 검색해보세요" />
           </LogoAndSearchBar>
           <LoginAndInfos>
-            {/* 추후제작될 버튼 컴포넌트 사용 예정 */}
             <RgbaBtn width="120px" height="36px" color="darkGray">
               파트너 등록하기
             </RgbaBtn>
-            <RgbaBtn width="70px" height="36px" color="darkGray">
-              로그인
-            </RgbaBtn>
-            <Button width="100px" height="36px" color="blue">
-              회원가입
-            </Button>
+            {isLogin === false ? (
+              <RgbaBtn
+                width="70px"
+                height="36px"
+                color="darkGray"
+                onClick={() => goToLogin()}
+              >
+                로그인
+              </RgbaBtn>
+            ) : (
+              <RgbaBtn
+                width="70px"
+                height="36px"
+                color="darkGray"
+                onClick={() => logout()}
+              >
+                로그아웃
+              </RgbaBtn>
+            )}
+            {isLogin === false ? (
+              <Button
+                width="100px"
+                height="36px"
+                color="blue"
+                onClick={() => goToSignUp()}
+              >
+                회원가입
+              </Button>
+            ) : (
+              <ProfileImg alt="profileimage" src={profileImg} />
+            )}
```

Comment:
- 두 컴포넌트가 한 쌍으로 움직여야 한다면 아래와 같이 바꿔보는건 어떨까요?
- 삼항연산자로 표현한다면 true / false 순이 되는게 자연스러워보여 순서를 바꿔보았습니다
- 여기도 즉시실행 함수보다는 그냥 함수를 넘겨주는게 좋아보이네요
```suggestion
            {isLogin ? (
              <>
                <RgbaBtn
                  width="70px"
                  height="36px"
                  color="darkGray"
                  onClick={logout}
                >
                  로그아웃
                </RgbaBtn>
                <ProfileImg alt="profileimage" src={profileImg} />
              </>
            ) : (
              <>
                <RgbaBtn
                  width="70px"
                  height="36px"
                  color="darkGray"
                  onClick={goToLogin}
                >
                  로그인
                </RgbaBtn>
                <Button
                  width="100px"
                  height="36px"
                  color="blue"
                  onClick={goToSignUp}
                >
                  회원가입
                </Button>
              </>
            )}
```